example/phpcs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<ruleset name="Project">
+    <rule ref="vendor/keboola/coding-standard/ruleset.xml"/>
```

Comment:
Zařve mi to:
```
phpcs -s -n --extensions=php application tests scripts library/Keboola
ERROR: Referenced sniff "vendor/keboola/coding-standard/ruleset.xml" does not exist
```
Asi by ten `ruleset.xml` mohl být spíš přímo v rootu než v `src` složce kde je samotný?

Reply:
Jo, sakra! Přesunul jsem ho a zapomněl upravit example. Je to proto, kdyby tam někdy byly custom sniffy v src, tak aby byl u nich.

Reply:
Fixed.